client/src/app/op.component.ts

```diff
@@ -32,5 +33,15 @@ import { ManagingBodiesComponent } from './+managing-bodies';
 ])
 
 export class OpAppComponent {
-  title = 'op works!';
+  title: string = 'op works!';
+
+  constructor(public translate: TranslateService) {
+      // use navigator lang if available
```

Comment:
Implements OnInit.
Import `OnInit` from `'@angular/core'`, add the `implements OnInit` interface, and move all this logic to ngOnInit [Style 09-01]. There shouldn't be any logic in the `constructor`.